src/View/Home/index.html.twig

```diff
@@ -1,5 +1,5 @@
 {% extends 'layout.html.twig' %}
 
 {% block content %}
-    <h1>Hello Wilder !</h1>
+    <h1 class = "title">Hello Wilder !</h1>
```

Comment:
No you don't seem to have a class title in your css so you shouldn't use it there.